synfig-studio/src/gui/cellrenderer/cellrenderer_value.cpp

```diff
@@ -172,6 +173,21 @@ class studio::ValueBase_Entry : public Gtk::CellEditable, public Gtk::EventBox
 		SYNFIG_EXCEPTION_GUARD_END_BOOL(true)
 	}
 
+	bool on_key_press_event(GdkEventKey* key_event)
+	{
+		SYNFIG_EXCEPTION_GUARD_BEGIN()
+		if(key_event->keyval == GDK_KEY_Escape)
+		{
+			hide();
+			if (parent) parent->grab_focus();
+			property_editing_canceled() = true;
+			Gtk::CellEditable::editing_done();
```

Comment:
The only thing I'd propose is to delete this namespace.
In case of an 'overrider' implementation of `editing_done()`, it won't be called :)

Reply:
@rodolforg take out ```Gtk::CellEditable``` namespace?

Reply:
@rodolforg Alright I removed ```Gtk::CellEditable``` namespace. Let me know if that is what you were thinking.